core/modules/user.py

```diff
@@ -683,7 +683,9 @@ def addSkel(self):
     def editSkel(self, *args, **kwargs):
         skel = super(User, self).editSkel().clone()
 
-        skel.password = PasswordBone(descr="Passwort", required=False)
+
+        skel.password.required=False
+
```

Comment:
This has nothing to do with the topic of the pull request!

Reply:
This was for an other PR,but I forgot to delete it.